junit-jupiter-api/src/main/java/org/junit/jupiter/api/DynamicContainer.java

```diff
@@ -107,4 +119,59 @@ public Stream<? extends DynamicNode> getChildren() {
 		return children;
 	}
 
+	public Optional<ExecutionMode> getDefaultChildExecutionMode() {
+		return Optional.ofNullable(defaultChildExecutionMode);
+	}
+
+	public sealed interface Configuration extends DynamicNode.Configuration<Configuration> {
```

Comment:
A tiny API documenation at least on the type (including a usage example) would be neat on a `public` type.